src/bindings/python/src/pyopenvino/pyopenvino.cpp

```diff
@@ -65,6 +66,8 @@ std::string get_version() {
 PYBIND11_MODULE(pyopenvino, m) {
     m.doc() = "Package openvino.pyopenvino which wraps openvino C++ APIs";
     m.def("get_version", &get_version);
+    m.def("get_batch", &ov::get_batch);
+    m.def("set_batch", &ov::set_batch);
```

Comment:
can we expose `set_batch(int64_t)` as well to allow `set_batch(f, 128)`?

Reply:
As I commented above, `size_t` should be ok in that case. Is `int64_t` required?

Reply:
int64_t is more preferable as in C++ user can use "-1" to indicate 'dynamic batch dimension'

Reply:
So it should defaults to `-1` as I assume?